src/web/components/Onwards/OnwardsLayout.tsx

```diff
@@ -6,22 +6,40 @@ import { Hide } from '@frontend/web/components/Hide';
 
 import { OnwardsTitle } from './OnwardsTitle';
 import { OnwardsContainer } from './OnwardsContainer';
+import { StoryPackage } from './StoryPackage';
 
 type Props = {
-    content: OnwardsType;
-    component: React.ElementType;
+    onwardSections: OnwardsType[];
 };
 
-export const OnwardsLayout = ({ content, component: Content }: Props) => (
-    <Flex>
-        <LeftColumn showRightBorder={false} showPartialRightBorder={true}>
-            <OnwardsTitle title={content.heading} />
-        </LeftColumn>
-        <OnwardsContainer>
-            <Hide when="above" breakpoint="leftCol">
-                <OnwardsTitle title={content.heading} />
-            </Hide>
-            <Content content={content.trails} />
-        </OnwardsContainer>
-    </Flex>
-);
+const decideLayout = (layout: OnwardsLayoutType, trails: TrailType[]) => {
+    switch (layout) {
+        case 'fourAndFour':
+            return <StoryPackage content={trails} />;
+        default:
+            return <></>;
+    }
+};
+
+export const OnwardsLayout = ({ onwardSections }: Props) => {
+    return (
+        <>
+            {onwardSections.map((onward, index) => (
+                <Flex key={`${onward.heading}-${index}`}>
+                    <LeftColumn
+                        showRightBorder={false}
+                        showPartialRightBorder={true}
+                    >
+                        <OnwardsTitle title={onward.heading} />
+                    </LeftColumn>
+                    <OnwardsContainer>
+                        <Hide when="above" breakpoint="leftCol">
```

Comment:
@SiAdcock might be interested in this

Reply:
Yeah, we use the same pattern [here](https://github.com/guardian/dotcom-rendering/blob/master/src/web/layouts/StandardLayout.tsx#L113) for the main article layout. I guess something similar will be all over fronts pages once we arrive there

You could defo build up a generic component with props for title, description, teaser image, etc. and then use `children` for the main content. I think this could be a case where an abstraction makes sense. Maybe just within DCR for now and think about promotion to SF later? @SiAdcock ?